fairscale/nn/data_parallel/sharded_ddp.py

```diff
@@ -118,52 +126,36 @@ def reduce(self) -> None:
         """
         assert self.module.training, "Cannot call reduce in eval"
 
-        def reduce_params(params: List[Parameter], params_rank: int) -> None:
```

Comment:
some of these cases were never used here, I assumed that they came from a copy-paste where this function was more generic than just reducing grads

Reply:
yeah, it was me mis-named it before.